quantum/quantum.c

```diff
@@ -27,6 +27,10 @@
 #define BREATHING_PERIOD 6
 #endif
 
+#ifndef MACRO_DELAY
```

Comment:
Could this be pulled out? 

It isn't exactly a small change.

Reply:
Yes! Sorry, I forgot I rebased this on #3092 for a fix